ios/RCTTWVideoModule.m

```diff
@@ -206,10 +207,17 @@ - (void)addParticipantView:(TVIVideoView *)view sid:(NSString *)sid trackSid:(NS
   if (self.localVideoTrack != nil) {
     [self.localVideoTrack setEnabled:enabled];
     resolve(@(enabled));
-  } else {
-    [self.localVideoTrack setEnabled:false];
-    resolve(@false);
   }
+  else{
+    [self createLocalVideoTrack];
+    resolve(@true);
+  }
+}
+
+RCT_EXPORT_METHOD(createLocalVideoTrack) {
```

Comment:
This doesn't need to be exported, since there's no call from JS land?

Reply:
Yeah. I forgot to change this. Will update the PR.

Reply:
Done.